internal/cli/get.go

```diff
@@ -386,9 +386,9 @@ func (o *GetOptions) printDevicesSummaryTable(w *tabwriter.Writer, summary *api.
 
 func (o *GetOptions) printDevicesTable(w *tabwriter.Writer, devices ...api.Device) {
 	if o.Output == wideFormat {
-		fmt.Fprintln(w, "NAME\tALIAS\tOWNER\tSYSTEM\tUPDATED\tAPPLICATIONS\tLAST SEEN\tLABELS")
+		_, _ = fmt.Fprintln(w, "NAME\tALIAS\tOWNER\tSYSTEM\tUPDATED\tAPPLICATIONS\tLAST SEEN\tVERSION\tLABELS")
```

Comment:
I don’t think the agent version is important enough to consume a column in the table output, not even in wide format.

Reply:
There is no other place to display it... Any ideas?

Reply:
It will be displayed with -oyaml, for example

Reply:
restored to original